docs/Node Operators/node-versions.md

```diff
@@ -12,13 +12,27 @@ metadata:
 
 You can find a list of release notes for Chainlink nodes in the [smartcontractkit GitHub repository](https://github.com/smartcontractkit/chainlink/releases). Docker images are available in the [Chainlink Docker hub](https://hub.docker.com/r/smartcontract/chainlink/tags).
 
-## Upgrading to v1.0.0 nodes
+## Changes to v1.1.0 nodes
+
+**[v1.1.0 release notes](https://github.com/smartcontractkit/chainlink/releases/tag/v1.1.0)**
+
+The v1.1.0 release includes several substantial changes to the way you configure and operate Chainlink nodes:
+
+- **Multichain support**: Chainlink now supports connecting to multiple different EVM chains simultaneously. See the [v1.1.0 Changelog](https://github.com/smartcontractkit/chainlink/blob/v1.1.0/docs/CHANGELOG.md#multichain-support-added) for details.
```

Comment:
probably should make note this is an alpha feature

Reply:
Marked as Beta. Closing this comment.